### ISSUE TITLE
Which pieces of code relating monolingual extractive summarization objective? 

### ISSUE THREAD

**Issue by jaydenluke** [zdou0830/crosslingual_summarization_semantic#2]
Hi there,

I am pretty new to programming. I am trying to find the pieces of code relating monolingual extractive summarization objective. Unfortunately, I can't find one.

Could you please kindly point out where should I look for especially in `word-level-supervised`.

Thank you very much.

**Comment by zdou0830** [comment]
Hi,

`beaver/model/nmt_model.py` contains code on predicting the extracted probability for each word/sentence.
`train.py` contains code on computing the loss.
`beaver/data/dataset.py` contains code on loading the scores from a pretrained monolingual extractive model.

**Comment by jaydenluke** [comment]
Thank you very much for your helpful answer!
